lib/router/mbaas-auth.js

```diff
@@ -0,0 +1,36 @@
+const q = require('q');
+
+/**
+ * Auth sends a request on 'wfm:user:auth' topic, raincatcher-demo-auth is subscriber
+ * of this topic and verifies passed in credentials.
+ *
+ * @param {Object} mediator object used to subscribe and publish read and auth topics
+ * @param {String} username Unique identifier of the user.
```

Comment:
@paolobueno But if a user decides that the username is not the unique identifier for the user (it could be email address etc) they can send something else as the identified (See https://github.com/feedhenry-raincatcher/raincatcher-demo-mobile/blob/master/src/app/auth/auth.js#L38)

This in turn calls https://github.com/feedhenry-raincatcher/raincatcher-user/blob/master/lib/user/user-client.js#L106 which is passed as the user Id

Therefore, `username` is not the correct term for the variable. It is the unique identifier for the user to authenticate by.